backend/app/main.py

```diff
@@ -0,0 +1,41 @@
+from fastapi import FastAPI
+from contextlib import asynccontextmanager
+
+from app.routers.records import router 
+from app.storage.storage import storage as wikipedia_storage
+
+import os
+
+@asynccontextmanager 
+async def lifespan(app: FastAPI):
+
+    """
+    A lifespan event to read in the Wikipedia output data and persist it in storage.
+    """
+        
+    file_path = os.getcwd()+"\\app\\data\\mini-wikipedia.output.txt"
```

Comment:
Use `__file__` - relative references instead, and `pathlib`. This code will only work on Windows.

Reply:
I am using `__file__` and `pathlib` to determine the output path now.